packages/common/mui/src/lib/action-sidebar/action-sidebar.tsx

```diff
@@ -16,7 +16,7 @@ import { NotionalTheme } from '@notional-finance/styles';
 
 export interface ActionSidebarProps {
   heading: MessageDescriptor;
-  helptext: MessageDescriptor & { values?: Record<string, unknown> };
+  helptext: any;
```

Comment:
Let's not use "any" here, the existing MessageDescriptor type should work right?

Reply:
No it would not work

Reply:
I can fuck with it some more but it was being a real pain and not worth the effort for this particular one in my opinion

Reply:
What is the cause of the type error? I dont think we should be using the `any` type in any core components like this